Hermes/appscale/hermes/__init__.py

```diff
@@ -219,22 +223,27 @@ def main():
 
     # Only master Hermes node handles /do_task route
     task_route = ('/do_task', TaskHandler)
+
+    if args.write_nodes_log:
+      stats_app.configure_node_stats_profiling()
+    if args.write_processes_log or args.write_detailed_processes_log:
+      stats_app.configure_processes_stats_profiling(
+        write_detailed_stats=args.write_detailed_processes_log)
+    if args.write_proxies_log or args.write_detailed_proxies_log:
```

Comment:
No, because all profile logs are saved on master node. And master node requests statistics from corresponding nodes.